storage/engine/mvcc.go

```diff
@@ -1182,7 +1182,8 @@ func MVCCResolveWriteIntent(engine Engine, ms *proto.MVCCStats, key proto.Key, t
 		}
 		// Get the bytes for the next version so we have size for stat counts.
 		value := proto.MVCCValue{}
-		ok, _, valueSize, err := engine.GetProto(kvs[0].Key, &value)
+		var valueSize int64
+		ok, _, valueSize, err = engine.GetProto(kvs[0].Key, &value)
```

Comment:
go-nyet seems to have a strong notion of shadowing: any time `:=` is used to define a variable already in scope, the new value is considered to shadow the old, which is verboten. Assignment via `=` is an explicit mutation, and is allowed. It's a bit clunky.

We could probably whitelist all the stuff we don't agree with and report issues upstream.